Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorTransformComponentSelection.cpp

```diff
@@ -1014,6 +1015,15 @@ namespace AzToolsFramework
         ToolsApplicationNotificationBus::Broadcast(&ToolsApplicationNotificationBus::Events::InvalidatePropertyDisplay, Refresh_Values);
     }
 
+    // leaves focus mode by focusing on the parent of the current perfab in the entity outliner
+    static void LeaveFocusMode()
+    {
+        if (auto prefabFocusPublicInterface = AZ::Interface<Prefab::PrefabFocusPublicInterface>::Get())
+        {
+            prefabFocusPublicInterface->FocusOnParentOfFocusedPrefab(GetEntityContextId());
+        }
```

Comment:
How does the UX for this feel when focusing on nested prefabs? From an X button I'd expect to leave focus mode altogether, but this will go one level up. Which may be the right behavior, I'm just worried the user may not notice?

Reply:
great Q, we should have a chat to see what makes sense in terms of how we want it to behave in these circumstances 👍